plugins/UM3NetworkPrinting/src/Messages/CloudFlowMessage.py

```diff
@@ -15,27 +15,23 @@
 
 class CloudFlowMessage(Message):
 
-    def __init__(self, address: str) -> None:
-
-        image_path = os.path.join(
-            CuraApplication.getInstance().getPluginRegistry().getPluginPath("UM3NetworkPrinting") or "",
-            "resources", "svg", "cloud-flow-start.svg"
-        )
+    def __init__(self, printer_name: str) -> None:
 
         super().__init__(
             text=I18N_CATALOG.i18nc("@info:status",
-                                    "Send and monitor print jobs from anywhere using your Ultimaker account."),
-            lifetime=0,
-            dismissable=True,
-            option_state=False,
-            image_source=QUrl.fromLocalFile(image_path),
-            image_caption=I18N_CATALOG.i18nc("@info:status Ultimaker Cloud should not be translated.",
-                                             "Connect to Ultimaker Digital Factory"),
+                                    f"Your printer <b>{printer_name}</b> could be connected via cloud.\n Manage your print queue and monitor your prints from anywhere connecting your printer to Digital Factory"),
+            title=I18N_CATALOG.i18nc("@info:title", "Are you ready for cloud printing?"),
+            message_type=Message.MessageType.CLOUD
         )
-        self._address = address
-        self.addAction("", I18N_CATALOG.i18nc("@action", "Get started"), "", "")
+        self._printer_name = printer_name
+        self.addAction("get_started", I18N_CATALOG.i18nc("@action", "Get started"), "", "")
+        self.addAction("learn_more", I18N_CATALOG.i18nc("@action", "Learn more"), "", "", button_style=Message.ActionButtonStyle.LINK, button_align=Message.ActionButtonAlignment.ALIGN_LEFT)
```

Comment:
Code style: Spaces around binary operators.

```suggestion
        self.addAction("learn_more", I18N_CATALOG.i18nc("@action", "Learn more"), "", "", button_style = Message.ActionButtonStyle.LINK, button_align = Message.ActionButtonAlignment.ALIGN_LEFT)
```